src/test/e2e-emulated/contracts/repeat-range.tact

```diff
@@ -0,0 +1,41 @@
+contract RepeatRange {
+    /// Effective range counter
+    effCounter: Int = 0;
+
+    /// To handle deployment
+    receive() {}
+
+    /// from -2^{256} to 0, including both ends
+    get fun testIgnoredRange(): Int {
+        let counter = 0;
+
+        repeat ((-pow(2, 255)) * 2) { counter += 1 }
+        repeat (-1) { counter += 1 }
+        repeat (0) { counter += 1 }
+
+        return counter; // 0, if things go right
```

Comment:
How about we compare with `0` internally and just return a boolean? Sandbox tests can be made more uniform this way